### PR TITLE
Fix example labels

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java [hunk 1]
```diff
@@ -199,6 +199,7 @@ public static void main(String[] args) throws Exception {
       String importAccountId = "<accountid_to_be_imported>";
 
       System.out.println("importAccountToEnterprise() result:");
+
```
Comment (padamstx) [inline modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java:202]: No need for a label when we'll be displaying the status code after the "end" comment as we are below.

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearchExamples.java [hunk 2]
```diff
@@ -49,33 +49,42 @@ public static void main(String[] args) throws Exception {
...
     try {
       String searchCursor = null;
+      System.out.println("search() result:");
 
       // begin-search
+
       SearchOptions searchOptions = new SearchOptions.Builder()
         .query("GST-sdk-*")
-        .fields(new java.util.ArrayList<String>(java.util.Arrays.asList("*")))
+        .fields(new java.util.ArrayList<>(java.util.Arrays.asList("*")))
```
Comment (padamstx) [inline modules/examples/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearchExamples.java:58]:
```suggestion
        .addFields("*")
```
For options fields that are lists, we generate an "addXXX()" method to add an element to the list for that field.   I think using these "addXXX()" methods in the examples makes it easier for users.

File: modules/examples/src/main/java/com/ibm/cloud/platform_services/usage_metering/v4/UsageMeteringExamples.java [hunk 3]
```diff
@@ -100,15 +102,17 @@ public static void main(String[] args) throws Exception {
 
             ReportResourceUsageOptions reportResourceUsageOptions = new ReportResourceUsageOptions.Builder()
                     .resourceId(resourceId)
-                    .resourceUsage(new java.util.ArrayList<ResourceInstanceUsage>(
+                    .resourceUsage(new java.util.ArrayList<>(
```
Comment (padamstx) [inline modules/examples/src/main/java/com/ibm/cloud/platform_services/usage_metering/v4/UsageMeteringExamples.java:105]:
```suggestion
                    .addResourceUsage(resourceInstanceUsageModel)
```

**Review by padamstx — APPROVED** [review]
Looks good.   I made a few suggestions for changes, but will approve now assuming those are addressed.

**Review by pyrooka — APPROVED** [review]

**Pull request merged**